cmake/FindHIP.cmake

```diff
@@ -471,7 +471,13 @@ macro(HIP_PREPARE_TARGET_COMMANDS _target _format _generated_files _source_files
                 INPUT "${custom_target_script_pregen}"
                 )
             set(main_dep DEPENDS ${source_file})
-            set(verbose_output "$(VERBOSE)")
+            if(CMAKE_GENERATOR MATCHES "Makefiles")
+                set(verbose_output "$(VERBOSE)")
+            elseif(HIP_VERBOSE_BUILD)
```

Comment:
Where is HIP_VERBOSE_BUILD defined? How is it set? Is this an auto-created variable used by Ninja?

Reply:
I just update the file. It was actually taken from findCUDA.cmake